src/pages/homepage/homepage.css

```diff
@@ -1,5 +1,5 @@
 .quiz-home-page {
-  justify-content: space-around;
+  justify-content: space-around !important;
```

Comment:
Try to solve with specificity, instead of an important tag

Reply:
done